engine/chainstate/client.go

```diff
@@ -48,13 +46,9 @@ type (
 		feeQuotes                bool                               // If set, feeUnit will be updated with fee quotes from miner's
 	}
 
-	// minercraftConfig is specific for minercraft configuration
-	minercraftConfig struct {
-		broadcastMiners []*minercraft.Miner // List of loaded miners for broadcasting
-		queryMiners     []*minercraft.Miner // List of loaded miners for querying transactions
-
-		apiType   minercraft.APIType      // MinerCraft APIType(ARC/mAPI)
-		minerAPIs []*minercraft.MinerAPIs // List of miners APIs
+	broadcastConfig struct {
+		protocol string
```

Comment:
Is the "protocol" necessary? ... since we support only "arc" 🤔

Reply:
"protocol" removed